pkgs/tools/system/ansible/default.nix

```diff
@@ -19,14 +19,10 @@ pythonPackages.buildPythonPackage rec {
   dontPatchELF = true;
   dontPatchShebangs = true;
 
-  propagatedBuildInputs = with pythonPackages; [
+  pythonPath = with pythonPackages; [
```

Comment:
I suppose I could leave them as `buildInputs` and just make sure they're in the executable wrappers....